src/components/cards.js

```diff
@@ -93,7 +93,12 @@ const RecipeCard = ({
         >
           <span
             sx={{
+              width: `100%`,
               fontSize: `3`,
+              display: `-webkit-box`,
+              '-webkit-box-orient': `vertical`,
+              '-webkit-line-clamp': `2`,
+              overflow: `hidden`,
```

Comment:
It's throwing some errors about using kebab-case here 
![image](https://user-images.githubusercontent.com/26583824/74475925-995e9f00-4e65-11ea-9628-463246f94e90.png)